config/local/config.yml

```diff
@@ -1,12 +1,24 @@
+# IMPORTANT: Replace all values within <angle_brackets> with your specific configuration details.
+# Do NOT leave any angle brackets in the final configuration.
 metrics:
   prometheus_address: "0.0.0.0:8001"
 coingecko:
   base_url: "https://pro-api.coingecko.com/api/v3/"
-  api_key: <api_key>
+  api_key: <api_key>  # e.g. "CG-1234567890abcdef1234567890abcdef"
   requests_per_minute: 500
   cache_refresh_interval: "5m"
 order_filler:
   worker_count: 10
+fund_rebalancer:
```

Comment:
I noticed the fund rebalancer config was never added to the sample config, so i put in a reference with dummy numbers